CHANGELOG.next.asciidoc

```diff
@@ -206,6 +206,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - The `_id` generation process for S3 events has been updated to incorporate the LastModified field. This enhancement ensures that the `_id` is unique. {pull}42078[42078]
 - Fix Netflow Template Sharing configuration handling. {pull}42080[42080]
 - Updated websocket retry error code list to allow more scenarios to be retried which could have been missed previously. {pull}42218[42218] 
+- Fix truncation of bodies in request tracing. {pull}42327[42327]
```

Comment:
I think it would be nice to include a bit of the "how" part, like ... "by limiting bodies to 10% of the maximum file size".

Reply:
Yep. Done as suggested.